pcmk/NovaCompute

```diff
@@ -240,6 +236,11 @@ nova_monitor() {
 
     pid=`nova_pid`
 
+    #nova is stopping - consider it is running as long as lockfile exists
```

Comment:
This shouldn't be necessary... the cluster only sends one action for a resource at a time.
Ie. it wont check the status of a resource while also stopping it